hack/kind-cluster-build.sh

```diff
@@ -22,20 +22,32 @@ source $ROOT/hack/lib.sh
 
 hack::ensure_kubectl
 hack::ensure_helm
+hack::ensure_kind
 
 usage() {
     cat <<EOF
 This script use kind to create Kubernetes cluster,about kind please refer: https://kind.sigs.k8s.io/
-Before run this script,please ensure that:
-* have installed docker
-* have installed kind and kind's version == v0.4.0
+Before run this script,please ensure that docker have been installed.
+
+Note that kind will merge config to your KUBECONFIG file, you could use a new KUBECONFIG file if you do not
+want to mutate the defaulting one:
+
+    export KUBECONFIG=${HOME}/.kube/kind-config-<cluster>
 
 Options:
        -h,--help               prints the usage message
        -n,--name               name of the Kubernetes cluster,default value: kind
        -c,--nodeNum            the count of the cluster nodes,default value: 6
        -k,--k8sVersion         version of the Kubernetes cluster,default value: v1.12.8
        -v,--volumeNum          the volumes number of each kubernetes node,default value: 9
```

Comment:
should we reduce the default num of the cluster too? 1 control plane and 3 workers are enough for most cases.

Reply:
good point, changed